docs/en/glossary/index.asciidoc

```diff
@@ -6,10 +6,10 @@
 :logstash-terms:       true
 :xpack-terms:          true
 
-include::{asciidoc-dir}/../../shared/versions/stack/{source_branch}.asciidoc[]
-include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
+include::{docs-root}/shared/versions/stack/{source_branch}.asciidoc[]
+include::{docs-root}/shared/attributes.asciidoc[]
 
-:es-repo-dir:          {docdir}/../../../../elasticsearch/docs/reference
+:es-glossary:          {elasticsearch-reference}/glossary.asciidoc
```

Comment:
I think in this context it makes me rethink the name of this attribute. Is it too late to rename it to something like "es-ref-root" or "es-ref-dir" or "es-reference-dir"? Then it's clearer that it's referring to a path, not to the book or its URL
```suggestion
:es-glossary:          {elasticsearch-reference}/glossary.asciidoc
```

Reply:
https://github.com/elastic/docs/pull/1316

Reply:
Thanks!